### PR TITLE
Nip13 support for more nostr event types

### PR DESCRIPTION
**Pull request by thesimplekid**
Changes events and functions that calls it to accept u16 difficulty target argument no add NIP13 support for other types of nostr events. #14 

Includes changing doc test relay from hard coded to env variable. #15 If you prefer the hard coded value let me know I'll change it back :).

### CODE REVIEW

**Comment by 0xtlt** [comment]
Strange the conflict, can you take a look ? @thesimplekid

File: src/events.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub struct EventPrepare {
 }
 
 impl EventPrepare {
-    /// get_content returns the content of the event to be signed
+    /// //get_content returns the content of the event to be signed
```
Comment (0xtlt) [inline src/events.rs:27]: Why did you put a comment here ?

**Review by 0xtlt — APPROVED** [review]

File: src/events.rs (hunk 1 above)
Comment (thesimplekid) [reply src/events.rs:27]: I thought I got a clippy error to do that, but I just removed it and ran clippy locally and its not needed

Comment (thesimplekid) [reply src/events.rs]: Just pushed a fix, Sorry about the extra CI runs

**Pull request merged**